docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-hints-autodiscover.asciidoc

```diff
@@ -92,7 +92,7 @@ https://github.com/elastic/elastic-agent/tree/main/deploy/kubernetes/elastic-age
 [discrete]
 == Configure hints autodiscovery
 
-To enable hints, you must add `hints.enabled: true` to the provider's configuration:
+To enable hints autodiscovery, you must add `hints.enabled: true` to the provider's configuration:
```

Comment:
That section if not in https://github.com/elastic/ingest-docs/pull/304/files. Is this coming from another PR? If so we should mention it as well in the description.

Reply:
Updated the desc, it was the https://github.com/elastic/ingest-docs/pull/380